playbooks/templates/k8s/namespace/01_matomo-config-map.yml.j2

```diff
@@ -224,6 +224,19 @@ data:
 
     [TrackerHostname]
     hostname = "{{ wai_hostname['ingestion'] }}.{{ wai_base_domain[item] }}"
+
+{% if item == 'wai-play' %}
+
+    [SuperUserOnlyRestrictions]
+    super_user_only_restrictions_enabled = 1
+
+    [RestrictAnonymousAccess]
+    allowed_requests[] = "module=Widgetize"
+    allowed_requests[] = "module=API&method=API.getMatomoVersion"
+    allowed_requests[] = "module=API&method=UsersManager.getTokenAuth"
+    allowed_referrers[] = "module=Widgetize&action=iframe&widget=1"    
+    
+{% endif %}    
```

Comment:
Questa configurazion c'era solo in play quindi, per ora, l'abbiamo gestita in questo modo.

Reply:
Possiamo procedere a metterla in tutti gli ambienti.